docs/flyte_fundamentals/registering_workflows.md

```diff
@@ -297,6 +296,14 @@ pyflyte --pkgs <dir1> --pkgs <dir2> package ...
 This is useful in cases where you want to register two different Flyte projects
 that you maintain in a single place.
 
+
+If you encounter a ``ModuleNotFoundError`` when packaging, ensure to include the correct source paths by using the `--source` option. For instance:
+
+```{prompt} bash $
+pyflyte --pkgs <dir1> package --source ./src -f
```

Comment:
@wild-endeavor 
I know the "old" flags still work but wanted to double-check that this command would still be valid even if we deprecate the old flags? (re https://github.com/flyteorg/flytekit/pull/2690)